cmd/func/main.go

```diff
@@ -38,7 +39,9 @@ func main() {
 		}}
 
 	if err := cmd.NewRootCmd(cfg).ExecuteContext(ctx); err != nil {
-		fmt.Fprintf(os.Stderr, "Error: %v\n", err)
+		if !errors.Is(err, terminal.InterruptErr) {
```

Comment:
If opposite is true (err is interrupt) we should return 130.

Reply:
see the suggestion

Reply:
+1 thanks, ^C should be 130 not 1